README.md

```diff
@@ -1,13 +1,29 @@
 <h1 align=center><img src=https://github.com/enBloc-org/kindly/assets/114364165/ec1855ea-13b7-4d61-800d-ff1f0c90ff70 height=200/></h1>
-Kindly is an open source web application created by Trafalgar Girls and enBloc, supporting Ukrainian refugees in the United Kingdom. Individuals can contribute items at no cost to provide much-needed support to members of the Ukrainian refugee community.
-<br>
-<br>
+
+Kindly is an open source web application created by Trafalgar Girls and <a href="https://github.com/enBloc-org">enBloc</a> that supports Ukrainian refugees in the United Kingdom. Individuals can contribute items at no cost to provide much-needed support to members of the Ukrainian refugee community.
+
+
+- ### Who is it for?
+
+   Kindly is for anyone who wants to share, sell or donate items to the Ukrainian refugee community. Equally, it is for members of the Ukrainian refugee community who want to borrow, buy or recieve items.
+
+- ### Who are we?
+
+   **Trafalgar Girls** is a volunteer project providing informational, practical and emotional support to Ukrainian Refugees living in Great Britain and beyond.
+   
+   **enBloc** is a web agency born from the ever growing Founders and Coders alumni community and a creator of open source tools. Our ethos is simple: inspire, collaborate, and grow together.
+
+
+- ### How to start using it?
+
+   Kindly is currently in beta stage, meaning that it is being tested by a small community of volunteer testers. If you want to be a part of the testing community, contact **XYZ**
```

Comment:
Should replace XYZ with maybe a link to the discord server? Or a trafalgar girls email? Not sure